### ISSUE TITLE
Documentation of Demo has python2 code examples

### ISSUE THREAD

**Issue by albertogomcas** [ipython/ipython#13672]
https://ipython.readthedocs.io/en/stable/api/generated/IPython.lib.demo.html

The example of demo.py uses print statements instead of print function, which does not work in python 3.x

**Comment by yashlad681** [comment]
Hey @albertogomcas mind if I work on this issue?

**Comment by MrMino** [comment]
@yashlad681 go for it if

**Comment by MrMino** [comment]
I don't think it was. As long as there's no PR sent - anyone is welcome to do so :).
